metadata-ingestion/src/datahub/ingestion/source/metabase.py

```diff
@@ -583,7 +629,7 @@ def get_datasource_from_id(self, datasource_id):
                 key=f"metabase-datasource-{datasource_id}",
                 reason=f"Unable to retrieve Datasource. " f"Reason: {str(http_error)}",
             )
-            return None, None
+            return None, None, None, None
```

Comment:
Please, can you add type annotations for `get_datasource_from_id` method ?
something on the lines of 
`def get_datasource_from_id(self, datasource_id:str)->Tuple[Optional[str],Optional[str],Optional[str],Optional[str]]:`